<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvoiceSetup</class>
 <widget class="QDialog" name="InvoiceSetup">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>642</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Setup Invoice</string>
  </property>
  <widget class="QComboBox" name="commissionDefaultCombo">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>10</y>
     <width>120</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Buyer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Company</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="showNumberBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Invoice# ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Height of invoice</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="invoiceHeight">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>68</number>
   </property>
   <property name="value">
    <number>45</number>
   </property>
  </widget>
  <widget class="QLabel" name="commissionDefaultLabel">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Comm Default:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="topSkip">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>35</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="showNameBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Names?</string>
   </property>
  </widget>
  <widget class="Q3GroupBox" name="checkOnInvoiceGroup">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>650</width>
     <height>120</height>
    </rect>
   </property>
   <property name="title">
    <string>Check On Invoice Options</string>
   </property>
   <widget class="QCheckBox" name="checkOnInvoice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>230</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Print Chec&amp;k On Invoice?</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="checkStartLine">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>64</number>
    </property>
    <property name="value">
     <number>45</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="prorateInvoiceCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>230</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Invoice Each Prorate?</string>
    </property>
   </widget>
   <widget class="QLabel" name="checkLineLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>First Check Line</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkPrinterOption">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>380</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Check this option to use the check printer for checks on invoices.&lt;BR&gt; If the check on invoice button is checked and no checks are printed,&lt;BR&gt; then the seller invoice printer is used.</string>
    </property>
    <property name="text">
     <string>Print Checks W/ Invoices On Check Printer</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="textLabel1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Skip Lines:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Q3Table" name="transTableSetup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>930</width>
     <height>88</height>
    </rect>
   </property>
   <property name="numRows" stdset="0">
    <number>1</number>
   </property>
   <property name="numCols" stdset="0">
    <number>11</number>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>50</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Min. Header Size:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="minHeaderSizeSpin">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_3_2_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Table Columns</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_3_2_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>50</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Into:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_3_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Force The Last:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="transOverrideColumnSpin">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>11</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_3_2_2_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>50</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Text Columns</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="transOverrideLengthSpin">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>50</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>96</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="Q3TextEdit" name="marketInvoiceComment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>480</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans Mono</family>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_2_2_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>450</y>
     <width>480</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Invoice Footer Comment</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="totalNSCheckBox">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>600</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>N/S in total</string>
   </property>
  </widget>
  <widget class="Q3TextEdit" name="marketInvoiceDisclaimer">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>480</y>
     <width>440</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans Mono</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="overwriteMode" stdset="0">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>600</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="brandOptionCheckBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>600</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Brands On Invoice</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="averageOptionCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Sex Summar&amp;y</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="averageOnLeftCheckBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>600</y>
     <width>240</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Overall Avg. / CWT On left</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="showTransactionCommentBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>260</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Transaction Comments?</string>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_2_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>480</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Invoice Header Comment</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="groupChargesBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>260</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Group Charges By Name</string>
   </property>
   <property name="shortcut">
    <string>Alt+G</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="showTagsAtBottomOfInvoiceCheckBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>321</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Show tag list at bottom of invoice</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="lpCopyCountSpinBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>350</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of copies (only for laser printer)</string>
   </property>
  </widget>
  <widget class="QLabel" name="laserPrinterChecksOnlyPrintAtLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Laser Printer, checks only. Print at:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxLaserCheckTop">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>390</y>
     <width>79</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxLaserCheckBottom">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>430</y>
     <width>79</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Bottom</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxLaserCheckMiddle">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>410</y>
     <width>79</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Middle</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linesPerCheck">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>410</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is normally 20. Calculated at 6 lines per inch, with 10 inches per page (two half inch margins, top and bottom). If it's not lining up for you, try changing this number by one and try again.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QLabel" name="linesPerCheckLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Lines per check:</string>
   </property>
  </widget>
  <widget class="QFrame" name="secondCopyFrame">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>320</y>
     <width>441</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QCheckBox" name="secondCopyPrinterOption">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Check this option to use a second printer for a copy of the invoice.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Print copies on Second Copy Printer</string>
    </property>
   </widget>
   <widget class="QLabel" name="checkLineCopyLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>First Check Line on copy: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="checkLineStartCopy">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>64</number>
    </property>
    <property name="value">
     <number>45</number>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends>Q3Frame</extends>
   <header>q3textedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>topSkip</tabstop>
  <tabstop>invoiceHeight</tabstop>
  <tabstop>checkOnInvoice</tabstop>
  <tabstop>checkStartLine</tabstop>
  <tabstop>transTableSetup</tabstop>
  <tabstop>saveButton</tabstop>
 </tabstops>
 <includes>
  <include location="local">dao/invoicedao.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
